packages/drivers/iframe-driver/src/outerDocumentServiceFactory.ts

```diff
@@ -43,6 +39,7 @@ const socketIOEvents = [
 
 export interface ICombinedDriver {
     clientId: string;
+    getResolvedUrl: () => Promise<IResolvedUrl>;
```

Comment:
>IResolvedUrl [](start = 34, length = 12)

i'd rather not see this change combined. the resolved url can container credentials so we need to be really careful about proxying it